src/api/IBMi.ts

```diff
@@ -1279,6 +1302,63 @@ export default class IBMi {
     return this.remoteFeatures[`QZDFMDB2.PGM`] !== undefined;
   }
 
+  getAllIAsps() {
+    return this.iAspInfo;
+  }
+
+  getIAspDetail(by: string|number) {
+    let asp: AspInfo|undefined;
+    if (typeof by === 'string') {
+      asp = this.iAspInfo.find(asp => asp.name === by);
+    } else {
+      asp = this.iAspInfo.find(asp => asp.id === by);
+    }
+
+    if (asp) {
+      return asp;
+    }
+  }
+
+  getIAspName(by: string|number): string|undefined {
+    let asp: AspInfo|undefined;
+    if (typeof by === 'string') {
+      asp = this.iAspInfo.find(asp => asp.name === by);
+    } else {
+      asp = this.iAspInfo.find(asp => asp.id === by);
+    }
+
+    if (asp) {
+      return asp.name;
+    }
```

Comment:
This looks a lot like `getIAspDetail`'s implementation. Reuse it then!
```suggestion
    return getIAspDetail(by)?.name;
```

Reply:
Smart!